src/Microsoft.Azure.WebJobs.Host/Bindings/ExecutionContext/ExecutionContextOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    public class ExecutionContextOptions
+    {
+        /// <summary>
+        /// The application directory to be used when creating <see cref="ExecutionContext"/> instances.
+        /// </summary>
+        public string AppDirectory { get; set; }
```

Comment:
The actual runtime directory is determined by the function indexer/loader right? I don't understand how this can be configurable like this. E.g. by default my functions are loading from Environment.CurrentDirectory, but I can set this to something other than that?

Reply:
I see now that you're just porting this over - I thought it was new

Reply:
No... all of this is existing logic, so I kept all of the existing properties.